src/twisted/python/test/test_release.py

```diff
@@ -912,15 +883,29 @@ def test_export(self):
         self.assertStructure(exportDir, structure)
 
 
+if os.name != "posix":
+    gitSkip, gitSkipText = True, "Release toolchain only supported on POSIX."
+else:
+    try:
+        gitVersion = runCommand(["git", "--version"]).split(b" ")[2].split(b".")
+    except FileNotFoundError:
+        gitSkip, gitSkipText = True, "git is not present."
+    else:
+        # We want git 2.0 or above.
+        if int(gitVersion[0]) >= 2:
+            gitSkip, gitSkipText = False, ""
+        else:
+            gitSkip, gitSkipText = True, "old git is present"
```

Comment:
I think this could be more clear

```python
def _gitSkip() -> str:
    if os.name != "posix":
        return "Release toolchain only supported on POSIX."
    try:
        gitVersion = runCommand(["git", "--version"]).split(b" ")[2].split(b".")
    except FileNotFoundError:
        return "git is not present"
    if int(gitVersion[0]) < 2:
        return "git 2.0 or above is required"
    return ""

gitSkipText = _gitSkip()
gitSkip = bool(gitSkipText)
```

Reply:
I  went with removing the whole git existence check.

I think that any env in which the tests are executed will have git...and will have a newer version of git.